tsfresh/transformers/feature_selector.py

```diff
@@ -89,6 +89,11 @@ def __init__(self, test_for_binary_target_binary_feature=defaults.TEST_FOR_BINAR
 
         :param chunksize: Size of the chunks submitted to the worker processes
         :type chunksize: int
+
+        :param ml_task: The intended machine learning task. Either `'classification'`, `'regression'` or `'auto'`.
+                    Defaults to `'auto'`, meaning the intended task is inferred from `y`.
+                    If `y` has a boolean or integer dtype, the task is assumend to be classification, else regression.
+        :type ml_task: str
```

Comment:
You are right, maybe we should add a method do "copy" documentation from the underlaying functions, for that we do not have to duplicate it all the time.
We could then maybe think about using **kwargs here.

But this is nothing for the next release or your PR ;-)